targets/XPlatCppSdk/make.js

```diff
@@ -275,9 +275,11 @@ function getRequestActions(tabbing, apiCall, isInstanceApi) {
             + tabbing + "    else if (request.authenticationContext->clientSessionTicket.length() > 0) {\n"
             + tabbing + "        authKey = \"X-Authorization\"; authValue = request.authenticationContext->clientSessionTicket;\n"
             + tabbing + "    }\n"
+            + tabbing + "    #if defined(ENABLE_PLAYFABSERVER_API) || defined(ENABLE_PLAYFABADMIN_API)\n"
```

Comment:
>     [](start = 25, length = 4)

Please remove these spaces, we usually put preprocessor directives at the very beginning of a line